stream-chat-android-client/src/main/java/io/getstream/chat/android/client/utils/observable/ChatObservable.kt

```diff
@@ -14,7 +15,7 @@ public interface ChatObservable {
     @Deprecated("Use the subscribe methods on ChatClient or ChannelController directly instead of events()")
     public fun first(): ChatObservable
     @Deprecated("Use the subscribe methods on ChatClient or ChannelController directly instead of events()")
-    public fun subscribe(listener: (ChatEvent) -> Unit): Subscription
+    public fun subscribe(listener: ChatClient.ChatEventListener<ChatEvent>): Subscription
```

Comment:
This method is deprecated already, no reason to change it - we'll remove it soon.